boleto/src/main/java/com/adyen/checkout/boleto/internal/ui/DefaultBoletoDelegate.kt

```diff
@@ -235,6 +235,7 @@ internal class DefaultBoletoDelegate(
             paymentMethod = GenericPaymentMethod(
```

Comment:
Should we make a specific payment method class per pm? I think `GenericPaymentMethod` is also barely used anywhere else

Reply:
I agree but let's not change them for existing components if they cause a breaking change